### PR TITLE
Users/ajay/added telemetry logs

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ async function run() {
         var errorMessage: string = "";
         var updateDeploymentStatus: boolean = true;
         var azureEndpoint: AzureEndpoint = await new AzureRMEndpoint(connectedServiceName).getEndpoint();
-
+        console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureAppServiceManage]" + '{"endpointId":"' + connectedServiceName + '"}');
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManage/azureappservicemanage.ts]: nit: Can we use JSON.stringify to create the JSON string for Xplat tasks?

Comment (rokmicrosoft) [inline Tasks/AzureAppServiceManage/azureappservicemanage.ts]: Please create variable for json just like created for AppService manage

File: Tasks/AzureMysqlDeployment/azuremysqldeploy.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ async function run() {
         azureMysqlTaskParameter = new AzureMysqlTaskParameter();
         task.debug('parsed task inputs');
         const endpoint: AzureEndpoint = await new AzureRMEndpoint(azureMysqlTaskParameter.getConnectedServiceName()).getEndpoint();
+        console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureMysqlDeployment]" + '{"endpointId":"' + azureMysqlTaskParameter.getConnectedServiceName() + '"}');
```
Comment (rokmicrosoft) [inline Tasks/AzureMysqlDeployment/azuremysqldeploy.ts]: Again create variable.

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts (hunk 1 above)
Comment (vincentdass) [reply Tasks/AzureAppServiceManage/azureappservicemanage.ts]: Also, please ensure that the tasks (modified here) has a higher patch version than Releases/M133

Comment (Ajay-MS) [reply Tasks/AzureAppServiceManage/azureappservicemanage.ts]: Yes it has.

**Pull request merged**